ozaria/site/components/play/PageUnitMap/index.vue

```diff
@@ -76,161 +73,161 @@
       }
       window.removeEventListener('resize', this.onWindowResize)
     },
-    methods: Object.assign(
-      {},
-      mapActions({
-        fetchCampaign: 'campaigns/fetch'
+    methods: {
+      ...mapActions({
+        fetchCampaign: 'campaigns/fetch',
+        setCourseInstanceId: 'layoutChrome/setCurrentCourseInstanceId'
       }),
-      {
-        playAmbientSound () {
-          const file = ((this.campaignData || {}).ambientSound || {})[AudioPlayer.ext.substr(1)]
-          if (!file || !me.get('volume') || this.ambientSound) {
-            return
-          }
-          const src = `/file/${file}`
-          if (!(AudioPlayer.getStatus(src) || {}).loaded) {
-            AudioPlayer.preloadSound(src)
-            Backbone.Mediator.subscribeOnce('audio-player:loaded', this.playAmbientSound, this)
-            return
-          }
-          this.ambientSound = createjs.Sound.play(src, { loop: -1, volume: 0.1 })
-          return createjs.Tween.get(this.ambientSound).to({ volume: 1.0 }, 1000)
-        },
-        async loadUnitMapData () {
-          try {
-            this.dataLoaded = false
-            this.levelSessions = await api.users.getLevelSessions({ userID: me.get('_id') })
-            await this.fetchCampaign(this.campaign)
-            this.campaignData = this.campaignDataByIdOrSlug(this.campaign)
-            if (this.courseInstanceId) {
-              this.levels = await this.buildLevelsDataForCourse()
-            } else {
-              // for anonymous/home users
-              this.levels = this.campaignData.levels
-            }
-            this.createLevelStatusMap()
-            this.determineNextLevel()
-            this.dataLoaded = true
-          } catch (err) {
-            console.error('ERROR:', err)
-            // TODO: update after a consistent error handling strategy is decided
-            noty({ text: 'Error in creating unit map data', layout: 'topCenter', type: 'error' })
+      playAmbientSound () {
```

Comment:
nit: newlines between entries make the code more readable by splitting up each method into its own visual section (aka paragraph).